boards/arm/raytac_mdbt53_db_40_nrf5340/Kconfig.defconfig

```diff
@@ -0,0 +1,98 @@
+# Raytac MDBT53-DB-40 nRF5340 board configuration
+
+# Copyright (c) 2019-2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if  BOARD_RAYTAC_MDBT53_DB_40_NRF5340_CPUAPP || BOARD_RAYTAC_MDBT53_DB_40_NRF5340_CPUAPP_NS
+
+config BOARD
+	default "raytac_mdbt53_db_40_nrf5340_cpuapp"
+
+# By default, if we build for a Non-Secure version of the board,
+# enable building with TF-M as the Secure Execution Environment.
+config BUILD_WITH_TFM
+	default n if BOARD_RAYTAC_MDBT53_DB_40_NRF5340_CPUAPP_NS
+
+if BUILD_WITH_TFM
+
+# By default, if we build with TF-M, instruct build system to
+# flash the combined TF-M (Secure) & Zephyr (Non Secure) image
+config TFM_FLASH_MERGED_BINARY
+	bool
+	default n
+
+endif # BUILD_WITH_TFM
```

Comment:
These lines do nothing, default to n is already the default.

TF-M support for this board in NCS will be added as a noup patch:
See https://github.com/nrfconnect/sdk-zephyr/commit/1f9e520d7518322c0cfd40d91f299cfa28ad07fb for how this has been done for other boards.

```suggestion
```